src/widgetresize/resizer.js

```diff
@@ -62,6 +62,14 @@ export default class Resizer {
 		 */
 		this._domResizerWrapper = null;
 
+		/**
+		 * View to a wrapper of an element controlled by this resizer.
+		 *
+		 * @private
+		 * @type {module:engine/view/element~Element|null}
+		 */
+		this._resizerWrapperView = null;
```

Comment:
The "view" doesn't mean that it's a view but it comes from the view so it's an adjective and should be the first word. Just like in the variable which you actually use below:

```js
this._resizerWrapperView = viewResizerWrapper;
```

Next to the `viewResizerWrapper` you have a `domResizerWrapper`.

Reply:
Please also fix the description of this property.